src/Docker.DotNet/Endpoints/StreamUtil.cs

```diff
@@ -30,30 +31,16 @@ internal static async Task MonitorStreamAsync(Task<Stream> streamTask, DockerCli
 
         internal static async Task MonitorStreamForMessagesAsync<T>(Task<Stream> streamTask, DockerClient client, CancellationToken cancel, IProgress<T> progress)
         {
+            var serializer = new Newtonsoft.Json.JsonSerializer();

```

Comment:
Can't we avoid Json.Net here and use System.Text.Json? 

I know this PR is only to get the tests back again. It is ok to keep it and we can deal with it on my PR later on for the next version.

However, I wouldn't create a new instance of it on every call of this method but instead, keep a shared instance (static) on this class.

Reply:
Should be already updated